src/WebUI/locales/en.yml

```diff
@@ -502,13 +503,31 @@ character:
         notify:
           warning: A large shield cannot be used on horseback
       upgrade:
-        upgradesTitle: Item Upgrades
+        upgradesTitle: "Item upgrades"
+        tooltip:
+          title: "Upgrade item"
+          description: "You can upgrade your item for a heirloom point"
+        validation:
+          maxRank: "The item already has a maximum rank"
+          loomPoints: "You don't have enough heirloom points"
+          exist: "You already have an improved version of this item"
         confirm:
-          description: 'Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}'
-          value: Upgrade item
+          description: "Are you sure you want to upgrade this item for {loomPoints}? Your item {oldItem} will be changed to {newItem}"
```

Comment:
Would this read as "Are you sure you want to upgrade this item for **{1}**? Your item **{+1 hoe}** will be changed to **{+2 hoe}**"? 
If yes:
Recommend change to -> Are you sure you want to upgrade this item for **{loomPoints} loompoint?** Your item **{oldItem}** will be changed to **{newItem}**